c2corg_ui/attributes.py

```diff
@@ -43,3 +43,21 @@
     'paragliding_takeoff',
     'paragliding_landing'
 ]
+
+# FIXME: incorrect values
```

Comment:
Yes, it's in the todo list. But I thought it could be done in a separated PR. But no problem doing it in this branch if you think it makes sense.